apps/cowswap-frontend/src/pages/Account/LockedGnoVesting/claimData/index.ts

```diff
@@ -10,16 +10,18 @@ interface Claim {
   proof: string[]
 }
 
-const indexFiles = {
+const indexFiles: Record<SupportedChainId, string[]> = {
   [SupportedChainId.MAINNET]: mainnetIndex,
   [SupportedChainId.GNOSIS_CHAIN]: gnosisChainIndex,
   [SupportedChainId.GOERLI]: goerliIndex,
+  [SupportedChainId.SEPOLIA]: [], // TODO SEPOLIA: check it
 }
 
-const chainNames = {
+const chainNames: Record<SupportedChainId, string> = {
   [SupportedChainId.MAINNET]: 'mainnet',
   [SupportedChainId.GNOSIS_CHAIN]: 'gnosisChain',
   [SupportedChainId.GOERLI]: 'goerli',
+  [SupportedChainId.SEPOLIA]: '', // TODO SEPOLIA: check it
```

Comment:
Would be good for testing, but this is no longer possible so probably fine to not have it.

Anyway, have you checked with Gnosis team whether they have this?
I believe they were the ones who wrote this contract.

Reply:
Asked: https://cowservices.slack.com/archives/C036MD8ADMG/p1704446589550619